benchmarks/yamauchi_takei_2016_anelasticity/anelasticity_temperature.h

```diff
@@ -0,0 +1,128 @@
+/*
+  Copyright (C) 2016 - 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_initial_temperature_anelasticity_temperature_h
+#define _aspect_initial_temperature_anelasticity_temperature_h
+
+#include <aspect/initial_temperature/interface.h>
+#include <aspect/utilities.h>
+#include <aspect/simulator_access.h>
+#include <cmath>
+#include <algorithm>
+#include <functional>
+
+namespace aspect
+{
+  namespace InitialTemperature
+  {
+    using namespace dealii;
+
+    /**
+     * A class that implements a prescribed temperature field determined from
+     * a AsciiDataProfile input file.
```

Comment:
I think you got this comment by copying from somewhere. Can you update it?

Reply:
Done